app/actions/pricesActions.js

```diff
@@ -26,25 +37,60 @@ function mapPrices(pricingData: Array<any>, currency) {
     },
     {},
   )
+
+  return prices
+}
+
+const apiCall = async url => {
+  const priceDataResponse = await axios.get(url)
```

Comment:
There have been cases in the past where the cryptocompare API has been down... I am concerned that that we will have issues here calling `Object.values(priceDataResponse.data.RAW)` (on a potentially null or undefined response if they are experiencing down time) 

Can we add some defensive programming here to account for that?